### PR TITLE
Flow Switch PPM Implementation

### PR DESCRIPTION
**Pull request by nrwslac**
## Description

- Made `fbFlowMeter` offset default to 0.
- Update PV name for flow switch FLOW->FSW per CCC doc
- Implement flow switch in `PPM`

## Motivation and Context

- Implementation of PCW flow switches
## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive comments
- [x] Test suite passes locally
- [x] Libraries are set to ``Always Newest`` version (``Library, *``)
- [x] Committed with ``pre-commit`` or ran ``pre-commit run --all-files``

### CODE REVIEW

File: lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU
Comment (ZLLentz) [inline lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU:170]: The implementation is good.
What's the context here? The PPM can have either a flow switch, a flow meter, or both? The previous fbFlowMeter is still present, so I'm probably just out of the loop. Do we need to have some sort of mechanism to decide which to show in the UI later?

Comment (ZLLentz) [reply lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU:170]: Or, should we be doing something in the PLCs to handle this split?

Comment (nrwslac) [reply lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU:170]: Yea, the former. The PPMs in the EBD/FEE have flow switches and the hutches have flow meters. We will need to deal with this somewhere; I thought on the GUI level, but open to suggestions.

File: lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU
Comment (ZLLentz) [reply lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU:98]: This is a correct assumption

Comment (ZLLentz) [reply lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU:98]: The result is probably PVs like `IM3L0:PPM:FSW_RBV`
Without the FSW -> FLOW change the PV we could have put the FSW here and had `IM3L0:PPM:FSW:FLOW_RBV`
I think either is probably fine, but maybe it's more correct for the CCC to have it's own colon-delimited section?

Probably all of these are suboptimal and we really wanted something like:
`IM3L0:PPM:FSW:FLOW_OK_RBV`?

I think there's some room for judgement calls here.

Note that any of these permutations don't seem to break any existing pcdsdevices, so from a "is this OK in code" standpoint they all pass

File: lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU
Comment (ZLLentz) [reply lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU:170]: I think I agree, we can make a sequel to PPMCool

File: lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU
Comment (ZLLentz) [reply lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU:98]: I'll approve after I hear your opinions on the naming, there's no reason to hold up the PR except to lock in the PV names

Comment (nrwslac) [reply lcls-twincat-common-components/Library/Devices/PPM/FB_PPM.TcPOU:98]: I really like the last thought `IM3L0:PPM:FSW:FLOW_OK_RBV`. Adds a bit of intuition, FSW_RBV is ambiguous.

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**
